### DISCUSSION TITLE
Versioning

### DISCUSSION THREAD

**Discussion by ka7eh** [NorESMhub/noresm-land-sites-platform#90] [General]
@lasseke @evalieungh @huitang-earth this is a continuation of a conversation around versioning here: https://github.com/NorESMhub/NorESM_LandSites_Platform/pull/88#issuecomment-1061558263.

Usually, versions below 1 are considered unstable, though it depends on the project.
Regardless, I agree with @lasseke that it's good to use a tag like `dev` to specify the state of the work explicitly.

Alternatively, we can use [Calendar Versioning](https://calver.org/) scheme, which uses dates instead of arbitrary numbers for versions. In our case, if for example, we go with short year and month, the first version will be `22.03.0-dev`.

I didn't mean to overcomplicate this but thought to have a brief conversation here and just pick a scheme.

So here are the candidates so far for the first release:
- `0.1.0-dev`
- `22.03.0-dev`

**Comment by evalieungh** [comment]
I think I have a slight preference for the conventional numbering rather than calendar. Numbered versioning seems more informative about what stage of development we are in and how many versions have preceded the current one (in case we ever get to v2,3,4 by the time we are all grey-haired).

**Comment by ka7eh** [comment]
Sounds good. Let's start with `0.1.0-dev` and we can change it down the road if needed.
